media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -4220,6 +4220,9 @@ VAStatus DdiMedia_DeriveImage (
         break;
     case Media_Format_X8R8G8B8:
     case Media_Format_X8B8G8R8:
+    case Media_Format_Y210:
```

Comment:
According to my understanding, Y210 is not 32 bits per pixel.

Reply:
@FurongZhang , In my understanding, Y210 is ordered as YUYV..., each YUYV stands for 2 pixel, and each channel has 16bit. so the per_pixel = 4*16/2 =32.